### PR TITLE
feat(pairing): update the pairing strings based on latest UX

### LINKED ISSUE #556: Finalize Pairing strings
Ref: https://github.com/mozilla/fxa-content-server/pull/6985/files#r259089308

The strings in this chart https://www.lucidchart.com/documents/edit/9a420c19-1e92-42a5-8eae-908a442c1044/0 need to be finalized

cc @ryanfeeley

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #556

cc @ryanfeeley

### CODE REVIEW

**Comment by vladikoff** [comment]
based on https://www.lucidchart.com/documents/edit/9a420c19-1e92-42a5-8eae-908a442c1044/0

File: packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div id="main-content" class="card pair-auth">
   <header>
     <h1 id="fxa-pair-auth-allow-header">
-      {{#unsafeTranslate}}Approve connections <small>to begin syncing</small>{{/unsafeTranslate}}
+      {{#unsafeTranslate}}Approve new device <small>for %(email)s</small>{{/unsafeTranslate}}
```
Comment (vbudhram) [inline packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache]: I wasn't able to test this physically but the mockups have email address twice in the screen. Once here and again below the location. Is this intended?

Screen Shot 2019-04-23 at 8 54 04 AM

File: packages/fxa-content-server/app/tests/spec/views/pair/auth_allow.js [hunk 2]
```diff
@@ -56,6 +56,9 @@ describe('views/pair/auth_allow', () => {
       window: windowMock,
     });
     broker.set('remoteMetaData', REMOTE_METADATA);
+    broker.set('browserSignedInAccount', {
```
Comment (vbudhram) [inline packages/fxa-content-server/app/tests/spec/views/pair/auth_allow.js]: You could probably add some asserts below that email address gets rendered in correct place.

File: packages/fxa-content-server/app/scripts/templates/pair/index.mustache [hunk 3]
```diff
@@ -6,17 +6,17 @@
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}">Ready the code scanner on your Firefox app.</div>
-    <p>Ready the code scanner on your Firefox app.</p>
+    <div class="graphic {{ graphicId }}">{{#t}}Open the code scanner on your Firefox app and show code when ready.{{/t}}</div>
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/scripts/templates/pair/index.mustache]: Seems a bit strange to have this text duplicated.

File: packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache (hunk 1 above)
Comment (shane-tomlinson) [reply packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache]: @vladikoff will this occur on other screens too?

Comment (vladikoff) [reply packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache]: @ryanfeeley do you want the text duplicated?

Comment (ryanfeeley) [reply packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache]: Oops! My bad guys! I removed it.
image

Comment (vladikoff) [reply packages/fxa-content-server/app/scripts/templates/pair/auth_allow.mustache]: Thanks @ryanfeeley  !

File: packages/fxa-content-server/app/scripts/templates/pair/index.mustache (hunk 3 above)
Comment (vladikoff) [reply packages/fxa-content-server/app/scripts/templates/pair/index.mustache]: gonna remove it

**Comment by vladikoff** [comment]
> @vladikoff will this occur on other screens too?

Just the two allow screens, based on the mockups

**Pull request merged**